tasks/manage_app.yml

```diff
@@ -1,12 +1,18 @@
 ---
-# Deploy an app
+# Task set to manage apps
```

Comment:
Can we call this file `manage_app_services.yml` instead?

Reply:
For me on app have a lot of services.
All over tasks who interact with app and services end with app on their names.

Reply:
I don't get your point: I want to move the `tasks/manage_app.yml` to `tasks/manage_app_services.yml` because "manage app" is too generic. Maybe "manage app services" is not relevant, but I have no better idea at the time. What about "manage app objects"?

Reply:
By definition one app is a stak : a collection of services and ressources.
This playbook manage one app, for me the name is good if we have a good doc.
Why do not rename `delete_app.yml` by `delete_app_objects.yml` if we rename this manage playbook ?

Reply:
Okay let's do this!